src/devices/LidarLiteV3/samples/README.md

```diff
@@ -0,0 +1,195 @@
+﻿# LidarLiteV3  LIDAR ToF Sensor by Garmin
+
+# Summary
+
+This device belongs to a class of sensors known as time-of-flight, which measures distances by 
+calculating the time delay between signal transmission and reception of the signal as it bounces 
+off the subject. Unlike the popular sonar-based HC-SR04, this device uses a low-power laser. The 
+advantage is longer distances (up to 40m) but more prone to errors when the subject is reflective. 
+
+# Raspberry PI Wiring
+
+The device communicates over I2C and while there's a PWN mode supported by the device, it is not 
+implemented.
+
+I2C Wiring on the Raspberry PI:
+
+| LidarLiteV3 Wire      | Raspberry PI GPIO Pin                   |
```

Comment:
add a note that this is physical numbering (most of the people use logical, logical is whatever BCM chip spec calls it)

Reply:
makes sense